src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -120,22 +120,30 @@ <h3 class="slds-section__title slds-theme_shade">
 
                                 </lightning-layout>
 
-                                <c-rd2-entry-form-schedule-section record-id={recordId} data-id="scheduleComponent"
-                                    onerrorevent={handleChildComponentError} ontypechange={handleRecurringTypeChange} if:true={isSettingReady}>
-                                </c-rd2-entry-form-schedule-section>
+                                <template if:true={isSettingReady}>
+                                    <c-rd2-entry-form-schedule-section 
```

Comment:
Changes to the schedule form and the Elevate credit card form more or less fully address ACs for edit commitment WIs: W-8585272 and W-8580238.
However, there is an issue with Elevate credit card form *not always* displaying for an existing Elevate RD. Not sure why, the schedule form was changed to dispatch an event when data is loaded so Recurring Type should be sent to the main RD entry form and the `isElevateWidgetEnabled` set to true.

Reply:
This WI is in synch with `main` and the create commitment order switch.